test/integration/password_reset_test.rb

```diff
@@ -1,19 +1,25 @@
 require 'test_helper'
+include ActiveJob::TestHelper
 
 class PasswordResetTest < SystemTest
   setup do
     @user = create(:user, handle: nil)
+    @old_queue_adapter = queue_adapter
+    ActiveJob::Base.queue_adapter = :test
```

Comment:
@simi 
> I think we can use test adapter as default one in test_helper.

I doubt we should tho. We don't always care if a job gets executed in test and using async adapter as default will make test faster.